bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/TombstoneEventEnhancer.kt

```diff
@@ -6,22 +6,46 @@ import androidx.annotation.RequiresApi
 import com.bugsnag.android.Thread as BugsnagThread
 
 internal class TombstoneEventEnhancer(
-    private val logger: Logger
+    private val logger: Logger,
+    private val listOpenFds: Boolean
 ) : (Event, ApplicationExitInfo) -> Unit {
     @RequiresApi(Build.VERSION_CODES.R)
     override fun invoke(event: Event, exitInfo: ApplicationExitInfo) {
         try {
-            TombstoneParser(logger).parse(exitInfo) { thread ->
-                mergeThreadIntoEvent(
-                    thread,
-                    event
-                )
-            }
+            val fileDescriptors = ArrayList<Map<String, Any>>()
+            TombstoneParser(logger).parse(
+                exitInfo, listOpenFds, { thread ->
+                    mergeThreadIntoEvent(
+                        thread,
+                        event
+                    )
+                },
+                { fd, path, owner ->
+                    fileDescriptors.add(
+                        mapOf(
+                            "fd" to fd,
+                            "path" to path,
+                            "owner" to owner,
+                        )
+                    )
+                    mergeFdToMetadata(
+                        fileDescriptors,
+                        event
+                    )
```

Comment:
This might be better done as (to improve the formatting in the Dashboard):
```suggestion
                    val fdInfo = if(owner.isNotEmpty()) mapOf("fd" to fd, "owner" to owner) else mapOf("fd" to fd)
                    event.addMetadata("Open FileDescriptors", path, fdInfo)
```